src/lexer/regexUtil.ts

```diff
@@ -4,6 +4,7 @@ import type { PrefixedQuoteType } from './regexTypes';
 export const escapeRegExp = (string: string) => string.replace(/[.*+?^${}()|[\]\\]/gu, '\\$&');
 
 export const WHITESPACE_REGEX = /\s+/uy;
+export const LINEBREAK_REGEX = /\v|\n|\r\n/g;
```

Comment:
A vertical tab? Really? If somebody uses the vertical tab character, they're really asking for trouble.

Also, we should match `\r\n` and `\n` in different order. Currently `\n` always matches first, so `\r\n` never matches.